tutorials/20-structured-data.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(table)
</pre><p>Use $table instead of print(table) for better output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>